coqpyt/tests/proof_file/utility.py

```diff
@@ -170,12 +178,19 @@ def add_step_defaults(step):
         step["context"] = []
 
 
-def get_test_proofs(yaml_file: str):
+def get_test_proofs(yaml_file: str, coq_version: Optional[str] = None):
```

Comment:
Why is the new argument optional?

Reply:
This is called in `test_changes.py` as well (lines 18, 30 and 96) where it doesn't depend on the version. This allows changes to be minimal.

Reply:
Alright, sounds good.